Maple2.Model/Error/MasteryError.cs

```diff
@@ -0,0 +1,18 @@
+﻿// ReSharper disable InconsistentNaming, IdentifierTypo
+
+using System.ComponentModel;
+
+namespace Maple2.Model.Error;
+
+public enum MasteryError : short {
+    [Description("Not enough mastery.")]
+    s_mastery_error_lack_mastery = 1,
+    [Description("Not enough mesos.")]
+    s_mastery_error_lack_meso = 2,
+    [Description("You have not completed the required quest.")]
+    s_mastery_error_lack_quest = 3,
+    [Description("Not enough items.")]
+    s_mastery_error_lack_item = 4,
+    [Description("Insufficient level.")]
+    s_mastery_error_invalid_level = 7,
+}
```

Comment:
Under Settings > General in Rider you can add these settings to add newlines at ends of files:
![image](https://user-images.githubusercontent.com/7710340/200775971-ceacd108-e71a-4bb8-82b8-c091f5c6277e.png)